### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by yorinasub17**

### CODE REVIEW

File: docs/_docs/02_testing-best-practices/alternative-testing-tools.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ nav_title_link: /docs/
 ## A list of infrastructure testing tools
 
 Below is a list of other infrastructure testing tools you may wish to use in addition to Terratest. Check out [How
-Terratest compares to other testing tools]({{site.baseurl}}/docs/alternative-testing-tools/how-terratest-compares-to-other-testing-tools/) to understand the trade-offs.
+Terratest compares to other testing tools]({{site.baseurl}}/docs/alternative-testing-tools#how-terratest-compares-to-other-testing-tools) to understand the trade-offs.
```
Comment (brikis98) [inline docs/_docs/02_testing-best-practices/alternative-testing-tools.md:16]: Shouldn't this be:

```suggestion
Terratest compares to other testing tools]({{site.baseurl}}/docs/testing-best-practices/alternative-testing-tools/#how-terratest-compares-to-other-testing-tools) to understand the trade-offs.
```

Comment (yorinasub17) [reply docs/_docs/02_testing-best-practices/alternative-testing-tools.md]: Ah yes that is right. Good catch!

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by yorinasub17** [comment]
Thanks for review! Merging it in now.

**Pull request merged**